internal/operators/nvidiagpu/nvidia_gpu_operator.go

```diff
@@ -193,3 +189,7 @@ func (o *operator) GetFeatureSupportID() models.FeatureSupportLevelID {
 func (o *operator) GetBundleLabels() []string {
 	return []string(Operator.Bundles)
 }
+
+func IsSupportedGpu(gpu *models.Gpu) bool {
+	return gpu.VendorID == nvidiaVendorID
```

Comment:
Do we support any NVidia GPU?
I know that consumer cards also have this vendor ID not just the data center versions.

Reply:
The NVIDIA documentation only talks about data center GPUs: https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/platform-support.html#supported-nvidia-data-center-gpus-and-systems . It doesn't say anything (positive or negative) about consumer cards.

We could improve this explicitly check the list of GPUs listed in that document. I am not sure if that is worth.

Anyhow, if we want to do that it should be in a different pull request. This pull request only moves the existing check to the `nvidiagpu` package. Please open a ticket if you believe this needs to be improved.

Reply:
I mean, I tested the GPU operator a while back and it does seem to work with desktop GPUs to some extent, I had it running on an RTX3080ti.

I suppose it's a question of supported vs not supported though, so maybe there should be a list of supported devices.